tests/log/set_enclave_log_level/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+add_subdirectory(host)
+
+if (BUILD_ENCLAVES)
+  add_subdirectory(enc)
+endif ()
+
+add_enclave_test(tests/set_enclave_log_level set_enclave_log_level_host
+                 set_enclave_log_level_enc)
+
+set_enclave_tests_properties(
+  tests/set_enclave_log_level
+  PROPERTIES
+  PASS_REGULAR_EXPRESSION
```

Comment:
Does this match the test output to the regular expression and then say the test has passed?

Reply:
The PASS_REGULAR_EXPRESSION will look for the strings in the stdout and will return failure if it doesn't find these strings. Since I am testing log outputs for various log_level, used this to test expected log strings